### PR TITLE
Move DurableTaskManager and extensions to TestCommon

### PR DESCRIPTION
**Pull request by AndersBallingPetersen**
## Description
https://app.zenhub.com/workspaces/mosaic-60a6105157304f00119be86e/issues/gh/energinet-datahub/team-mosaic/343

## Quality

- [x] Documentation is updated
- [x] Release notes are updated
- [x] Package version is updated
- [x] Public types and methods are documented
- [x] Tests are implemented and executed locally

### CODE REVIEW

**Review by dstenroejl — COMMENTED** [review]
Commented, but let's talk so I can explain.

File: source/TestCommon/source/FunctionApp.TestCommon/Azurite/AzuriteManager.cs [hunk 1]
```diff
@@ -128,6 +128,39 @@ public void StartAzurite()
         StartAzuriteProcess();
     }
 
+    /// <summary>
+    /// Cleanup Azurite storage.
+    /// </summary>
+    public static void CleanupAzuriteStorage()
```
[the remark this answers is not in the document]
Comment (AndersBallingPetersen) [reply source/TestCommon/source/FunctionApp.TestCommon/Azurite/AzuriteManager.cs:134]: You are right. As long as we have an instance of the AzuriteManager - static does not make sense.

[the remark this answers is not in the document]
Comment (AndersBallingPetersen) [reply source/TestCommon/source/FunctionApp.TestCommon/Azurite/AzuriteManager.cs:134]: You are right. :)

**Review by dstenroejl — APPROVED** [review]
LGTM

**Pull request merged**